app/views/bikes/edit_drivetrain.html.haml

```diff
@@ -1,96 +1,96 @@
 = form_for @bike, multipart: true, html: { class: 'primary-edit-bike-form' } do |f|
   .form-well-container.container{ class: "edit-bike-page-#{@edit_template}" }
     .row
-      = render partial: '/bikes/edit_primary_menu'
+      = render partial: 'bikes/edit_primary_menu'
       .col-md-8.form-well
         .form-wrap
           .form-well-form-header-always-visible
             %h3
-              Wheels
+              = t(".wheels")
 
           - %w(front rear).each do |side|
             .related-fields
               .form-group.row
                 %label.form-well-label{ for: "#{side}_standard" }
-                  #{side.titleize} wheel
+                  = t(".sidetitleize_wheel", sidetitleize: side.titleize)
                 .form-well-input.fancy-select-placeholder.unfancy
                   .standard-diams{ id: "#{side}_standard"}
-                    // ignore this select in dirty forms
+                    -# ignore this select in dirty forms
                     %select.form-control.dirtyignore
                       %option{ value: '' }
-                        Choose diameter
+                        = t(".choose_diameter")
```

Comment:
I think you can remove this translation section, because it will be dealt with in the helper functionality.